lib/prompt-app-definition.js

```diff
@@ -1,16 +1,24 @@
+const chalk = require('chalk');
 const inquirer = require('inquirer');
 const path = require('path');
 
 module.exports = async function promptAppDefinition() {
+  console.log(
+    chalk.dim(`
+NOTE: This will create an app definition in your Contentful organization.
+      - Read more about app definitions: ${chalk.underline('https://ctfl.io/app-definitions')}
+      - Read more about app locations: ${chalk.underline('https://ctfl.io/app-locations')}
+  `)
+  );
+
   const appDefinitionSettings = await inquirer.prompt([
     {
       name: 'name',
       message: `App name (${path.basename(process.cwd())}):`
     },
     {
       name: 'locations',
-      message:
-        'Select where your app can be rendered. \n NOTE: Read more about app locations at https://ctfl.io/app-locations\n',
+      message: `Select where your app can be rendered in:`,
```

Comment:
"can be rendered in" sounds  strange to me. I think "can be rendered" would be fine.